playbook/app/pb_kits/playbook/pb_copy_button/_copy_button.tsx

```diff
@@ -17,6 +18,7 @@ type CopyButtonProps = {
   tooltipPlacement?:  "top" | "right" | "bottom" | "left",
   tooltipText?: string,
   value?: string,
+  variant?: 'icon' | null,
```

Comment:
Generally our pattern is that if the default variant does not have a specific name, we call it 'default' as opposed to passing null as a variant value!